src/applications/coronavirus-chatbot/tests/01-first-question.e2e.spec.js

```diff
@@ -0,0 +1,40 @@
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts');
+
+const vaAvatarMatcher = 'div#webchat div.webchat__imageAvatar__image';
+const webchatBubbleContent = 'div.webchat__bubble__content';
+const covid19PreventionButton =
+  "div#webchat button[aria-label='COVID-19 prevention']";
+const benefitsAndClaimsButton =
+  "div#webchat button[aria-label='Benefits and claims']";
+const wearMaskButton = "div#webchat button[aria-label='Should I wear a mask?']";
+const responseBubble =
+  "//div[@id='webchat']//div[@class='webchat__bubble__content']//strong[contains(text(), 'What question')]";
+const coronavirusChatbotPath = '/coronavirus-chatbot/';
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  client
+    .openUrl(`${E2eHelpers.baseUrl}${coronavirusChatbotPath}`)
+    .waitForElementVisible('body', Timeouts.verySlow)
+    .assert.title('VA coronavirus chatbot | Veterans Affairs')
+    .waitForElementVisible(vaAvatarMatcher, 45000)
+    .assert.containsText(webchatBubbleContent, 'Before we get started')
+    .click(covid19PreventionButton)
+    .assert.isDisabledElement(covid19PreventionButton, true)
+    .assert.isDisabledElement(benefitsAndClaimsButton, true)
+    .useXpath()
+    .waitForElementVisible(responseBubble, Timeouts.normal)
+    .assert.containsText(
+      responseBubble,
+      'What question can we answer for you first?',
+    )
+    .useCss()
+    .assert.isDisabledElement(wearMaskButton, false)
+    .click(wearMaskButton)
+    .assert.isDisabledElement(wearMaskButton, true)
+    .axeCheck('.main')
+    .end();
+});
+
+// Note: This test requires the real API. Run locally with yarn watch --env.api=https://dev-api.va.gov
+module.exports['@disabled'] = true;
```

Comment:
is it possible to mock the API so this E2E test can be run on the CI/CD server?

Reply:
Unfortunately no, unless the API really communicates with Microsoft and provides a valid token, we won't be able to render a real healthbot chat which is required for this test. It's still helpful for locally testing if updates to our code or botframework break the chatbot, so we wanted to have it despite not being able to run on CI.